x/interchainstaking/keeper/keeper.go

```diff
@@ -578,7 +578,7 @@ func (k *Keeper) GetAggregateIntentOrDefault(ctx sdk.Context, z *types.Zone) (ty
 	var filteredIntents types.ValidatorIntents
 
 	if len(z.AggregateIntent) == 0 {
-		intents = k.DefaultAggregateIntents(ctx, z.ChainId)
+		intents = k.DefaultAggregateIntents(ctx, z.ID())
```

Comment:
Assuming this iterates over all validators and gives them an equal distribution, I expect this ought to be ChainID()